src/harness/testcases/WINNF_FT_S_SIQ_testcase.py

```diff
@@ -206,6 +206,127 @@ def test_WINNF_FT_S_SIQ_8(self):
+    self._sas_admin.InjectUserId({'userId': device_3['userId']})
+    self._sas_admin.InjectUserId({'userId': device_4['userId']})
+    self._sas_admin.InjectUserId({'userId': device_5['userId']})
+    self._sas_admin.InjectUserId({'userId': device_6['userId']})
+
+    # send registration requests
+    devices = [device_1, device_2, device_3, device_4, device_5, device_6]
+    request = {'registrationRequest': devices}
+    response = self._sas.Registration(request)
+
+    # Check registration response
+    cbsd_ids = []
+    for resp in response['registrationResponse']:
+      self.assertEqual(resp['response']['responseCode'], 0)
+      cbsd_ids.append(resp['cbsdId'])
+    del request, response
+
+    # Send Spectrum Inquiry request for six cbsds
+    # create inquiry with 6 requests
+    #
+    # the 1st inquiry is valid
+    spectrum_inquiry_0 = json.load(
+        open(os.path.join('testcases', 'testdata', 'spectrum_inquiry_0.json')))
+    spectrum_inquiry_0['cbsdId'] = cbsd_ids[0]
+
+    # the 2nd inquiry has one one parameter invalid
+    spectrum_inquiry_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'spectrum_inquiry_0.json')))
+    # cause invalid parameters by changing frequencies
+    spectrum_inquiry_1['inquiredSpectrum'][0]['lowFrequency'] = 3650000000.0
+    spectrum_inquiry_1['inquiredSpectrum'][0]['highFrequency'] = 3560000000.0
+    spectrum_inquiry_1['cbsdId'] = cbsd_ids[1]
+
+    # the 3rd inquiry has highFrequency parameter in inquiredSpectrum object is missing
+    spectrum_inquiry_2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'spectrum_inquiry_0.json')))
+    del spectrum_inquiry_2['inquiredSpectrum'][0]['highFrequency']
+    spectrum_inquiry_2['cbsdId'] = cbsd_ids[2]
+
+    # the 4th inquiry has lowFrequency parameter in inquiredSpectrum object is missing
+    spectrum_inquiry_3 = json.load(
+        open(os.path.join('testcases', 'testdata', 'spectrum_inquiry_0.json')))
+    del spectrum_inquiry_3['inquiredSpectrum'][0]['lowFrequency']
+    spectrum_inquiry_3['cbsdId'] = cbsd_ids[3]
+
+    # the 5th inquiry has inquiredSpectrum object completely missing
+    spectrum_inquiry_4 = json.load(
+        open(os.path.join('testcases', 'testdata', 'spectrum_inquiry_0.json')))
+    del spectrum_inquiry_4['inquiredSpectrum']
+    spectrum_inquiry_4['cbsdId'] = cbsd_ids[4]
+
+    # the 6th inquiry has the cbsdId missing
+    spectrum_inquiry_5 = json.load(
+        open(os.path.join('testcases', 'testdata', 'spectrum_inquiry_0.json')))
+
+    request = {'spectrumInquiryRequest': [spectrum_inquiry_0, spectrum_inquiry_1,
+        spectrum_inquiry_2, spectrum_inquiry_3, spectrum_inquiry_4, spectrum_inquiry_5]}
+    response = self._sas.SpectrumInquiry(request)['spectrumInquiryResponse']
+
+    # Check Spectrum Inquiry Response
+    # the 1st object will be response code 0
+    self.assertEqual(response[0]['response']['responseCode'], 0)
+    self.assertTrue('cbsdId' in response[0])
```

Comment:
No need for this, I think, since you check the cbsdId on line 298 (it will give an error if the field is completely missing).